layouts/partials/fragments/search.html

```diff
@@ -0,0 +1,76 @@
+{{- $bg := .Params.background | default "light" -}}
+{{- $self := . -}}
+{{- .page_scratch.Add "js" ("syna-search.js" | relLangURL) }}
+
+{{ "<!-- Pricing -->" | safeHTML }}
+<section id="{{ .Name }}">
+  <div class="overlay container-fluid
+    {{- printf " bg-%s" $bg -}}
+  ">
+  <div class="container py-5">
+    {{- with .Params.title }}
+      <div class="col text-center
+        {{- if or (eq $bg "white") (eq $bg "light") (eq $bg "secondary") (eq $bg "primary") -}}
+          {{- printf " text-%s" "dark" -}}
+        {{- else -}}
+          {{- printf " text-%s" "light" -}}
+        {{- end -}}
+        {{- if and (not (isset $self.Params "subtitle")) (not $self.Content) -}}
+          {{- printf " pb-4" -}}
+        {{- end -}}
+      ">
+        <h2>
+          {{- . | markdownify -}}
+        </h2>
+      </div>
+    {{- end -}}
+    {{- with .Params.subtitle -}}
+      <div class="row">
+        <div class="col text-center pt-2 pb-4
+          {{- if or (eq $bg "secondary") (eq $bg "primary") -}}
+            {{- printf " text-%s" "dark" -}}
+          {{- else -}}
+            {{- printf " text-muted text-%s" "secondary" -}}
+          {{- end -}}
+        ">
+          <h6>
+            {{- . | markdownify -}}
+          </h6>
+        </div>
+      </div>
+    {{- end }}
+    <div class="row">
+      <form action="{{ "search" | absURL }}" class="w-100">
+        <div class="col-md-6 col-12">
+          <div class="form-group row">
+            <label for="search-query" class="col-2 pl-0 pt-2">Search:</label>
```

Comment:
Wouldn't provide a label as we already have `title` and `subtitle`.

A few other design ideas:
* center by default
* possibility for left/right align
* use search icon similar to navbar
* match color/style (needs separate discussion)